scripts/game_structure/windows.py

```diff
@@ -2233,7 +2233,8 @@ def __init__(self, source_screen: "Screens"):
             },
         )
         self.text_block.rebuild_from_changed_theme_data()
-
+    
+    
```

Comment:
Why are these added? I'd advise removing them. Changes to unrelated files should be avoided.

Reply:
will remove them!